bridges/zap/go.mod

```
@@ -0,0 +1,25 @@
+module zap
```

This module name has a high chance of conflicting with the package it is trying to bridge with (https://pkg.go.dev/go.uber.org/zap). We should try to define this so users won't need to rename the import.

```suggestion
module otelzap
```

This will need additional changes other than this to fully migrate.